config/config.go

```diff
@@ -183,7 +184,8 @@ type DefaultStrategyConfiguration struct {
 }
 
 type ExponentialBackoffStrategyConfiguration struct {
-	RetryLimit uint64 `json:"retryLimit" envconfig:"CONVOY_RETRY_LIMIT"`
+	RetryLimit   uint64   `json:"retryLimit" envconfig:"CONVOY_RETRY_LIMIT"`
```

Comment:
Let's not mix both camel case and snake case
```suggestion
	RetryLimit   uint64   `json:"retry_limit" envconfig:"CONVOY_RETRY_LIMIT"`
```